plugins/im-export-breakpoints/_include-media-export-breakpoints.scss

```diff
@@ -0,0 +1,24 @@
+@function im-breakpoints-to-json($target-name) {
+  $json: '';
+  $target-value: map-get($breakpoints, $target-name);
+
+  @each $name, $value in $breakpoints {
+    $json: if($json == '', '{', $json + ',');
+    $state: $target-value >= $value;
+    $json: $json + '"#{$name}":#{$state}';
+  }
+
+  $json: $json + '}';
+
+  @return $json;
+}
+
+@mixin im-export-breakpoints($element: body) {
```

Comment:
> Import includeMedia.js onto your project. This file is just a simplistic approach to access the information sent across by include-media. Feel free to extend it to fit your needs.

I think it's a bit difficult to add more information on how to include the file without making assumptions about the JS architecture being used. What are your thoughts?